classes/AnyPage.php

```diff
@@ -32,8 +44,8 @@ public function save() {
 		$site = elgg_get_site_entity();
 
 		$this->access_id = ACCESS_PUBLIC;
-		$this->container_guid = $site->getGUID();
-		$this->owner_guid = $site->getGUID();
+		$this->container_guid = $site->guid;
+		$this->owner_guid = $site->guid;
```

Comment:
We have decided between ourselves that `->guid` is preferred by core for better readability, plus it saves a method call, reduces the risk of fatal errors.